shared/js/dialer/keypad.js

```diff
@@ -1,8 +1,9 @@
 /* exported KeypadManager */
```

Comment:
Has anyone specified whether we need to support speed dial from the emergency app?  I suppose someone could put 911 in as a speed dial (or maybe an ICE contact?)  If we don't have an answer right now, we could have this be a follow up item.